web/src/app/auth/login/page.tsx

```diff
@@ -80,6 +93,9 @@ const Page = async () => {
             )}
           </div>
         </div>
+        <div className="fixed bottom-4 right-4 z-50 text-slate-400 p-2">
+          VERSION w{ web_version } b{ backend_version }
```

Comment:
Let move it to a new section on the bottom of the admin page

<img width="573" alt="image" src="https://github.com/danswer-ai/danswer/assets/32520769/34e1e6ec-d723-45b3-8403-e403ba1b6eaa">

Reply:
If you want, you could also add some text saying "Danswer MIT Version" somewhere on the page